From 308aa92ee48d79bc1c1d18879673ec6951499bce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Fri, 7 Aug 2026 18:12:00 +0200
Subject: [PATCH 3/4] Fix Windows DLL install path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Michał Górny <mgorny@gentoo.org>
---
 cmake/shared.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/shared.cmake b/cmake/shared.cmake
index b6f4aea..455745b 100644
--- a/cmake/shared.cmake
+++ b/cmake/shared.cmake
@@ -7,6 +7,7 @@ endif()
 
 install(TARGETS libddwaf_shared EXPORT libddwaf-config
     DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 if(LINUX)
